README.md

```diff
@@ -1,2 +1,37 @@
 # programapi
-Program API
+
+This project downloads, processes, saves, and serves the static JSON files containing details of accepted speakers and submissions via an API.
+
+What this project does step-by-step:
+
+1. Downloads the Pretalx speaker and submission data, and saves it as JSON files.
+2. Transforms the JSON files into a format that is easier to work with and OK to serve publicly. This includes removing unnecessary/private fields, and adding new fields.
+3. Serves the JSON files via an API.
+
+## Installation
+
+1. Clone the repository.
+2. Install the dependency management tool: ``make deps/pre``
+3. Install the dependencies: ``make deps/install``
+4. Set up ``pre-commit``: ``make pre-commit``
+
+## Configuration
+
+You can change the event in the [``config.py``](src/config.py) file. It is set to ``europython-2024`` right now.
+
+## Usage
+
+- Run the whole process: ``make all``
+- Run only the download process: ``make download``
+- Run only the transformation process: ``make transform``
+
+**Note:** Don't forget to set the ``PRETALX_TOKEN`` environment variable before running the download process. And please don't make too many requests to the Pretalx API, it might get angry 🤪
```

Comment:
The most common pattern to set environment variable is to do it in the command line like `PRETALX_TOKEN=blablabla`, however this is bad, because this line is saved to the shell history. What about we give instructions to define the variable in a file and source it? or make the script itself rely on the file with secret?

Reply:
We can look for a .env file first, then the other option. This would probably be easier than telling people to source it. 

Though I'm open to just telling people to source it and not change the code, if you have a strong preference for that.

Reply:
Implemented it